doc/source/migration_v2.rst

```diff
@@ -19,3 +19,5 @@ Migration to v2 version
     The following is not an invalid morphology anymore:
     - 2 point soma
     - non-sequential ids
+- script ``morph_check`` and ``morph_stats`` changed to ``neurom check`` and ``neurom stats``
```

Comment:
this is used in morph_service i think, be careful here (@jdcourcol knows more on that)